tests/integration/test_rm_file.py

```diff
@@ -377,6 +389,17 @@ def test_rm_pushed_links(runner: CliRunner, project_id, tmpdir) -> None:
         runner,
         ["--debug", "add", "hello2.txt"]
     )
+    # should fail since the target file does not exist
+    run(
+        runner,
+        ["--debug", "add", "hello3.txt"],
+        expected_exit_code=1
+    )
+    # should fail since the target file does not exist
+    run(
+        runner,
+        ["--debug", "add", "hello4.txt"]
```

Comment:
should we change all references from `hello2.txt` to `hello4.txt` below as well?

Reply:
Also tried to drill down into why, but despite the `validate_document_in_elastic` used by all `test_rm_files.py` tests, the elastic record still shows up on the explorer page. This is not unique to symlinks, as seen below. Are you able to reproduce this as well?

```bash
g3t --profile cbds-dev init cbds-qww_1 --approve
g3t add bonk.txt 
g3t meta init
g3t commit -m "bonk"
g3t push
g3t rm bonk.txt 
g3t meta init
g3t commit -m "rm bonk"
g3t push --overwrite   # overwrite needed
```

Reply:
Updated the original comment, thanks for clarifying

Reply:
Great thanks I see the test changes reflected in there. Also, deleting a single document reference generates a Bundle as expected.

Added to the [docs checklist above](https://github.com/ACED-IDP/gen3_util/pull/122#pullrequestreview-2808138878), but we should explain to users that a Bundle.ndjson should be deleted after a successful push.